x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/action_bar/action_bar.tsx

```diff
@@ -150,7 +152,7 @@ export const ActionBar = ({
                       size="s"
                       color="success"
                       iconType="play"
-                      disabled={!isValid || isTestRunInProgress}
+                      disabled={!isValid || isTestRunInProgress || !hasServiceManagedLocation}
```

Comment:
Ideally, it would be nice to have popover content for why its disabled in this case.